gems/aws-sdk-resources/bin/aws-v3.rb

```diff
@@ -107,10 +107,11 @@ def env_bool key, default
 require 'aws-sdk-resources'
 
 # Finally load all of the gems. Core is loaded a second time because of STS.
```

Comment:
Is this comment still relevant? Also I think this gem should be bumped too and it probably needs a minimum core version bump because of that.